src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,61 @@
-export const Pagination = () => {};
+import { PaginationItem } from '../PaginationItem';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  setCurrentPage: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  setCurrentPage,
+}) => {
+  const itemsPerPage = Math.ceil(total / perPage);
+
+  const pages = Array.from({ length: itemsPerPage }, (_, i) => i + 1);
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={`${currentPage === 1}`}
+          onClick={() => currentPage !== 1 && setCurrentPage(currentPage - 1)}
```

Comment:
Adding an `href` attribute with '#prev' will cause the page to jump to the top when clicked. To prevent the default navigation behavior, you should call `event.preventDefault()` in the `onClick` handler.